pom.xml

```diff
@@ -927,15 +873,4 @@ flexible messaging model and an intuitive client API.</description>
       <url>http://yahoo.bintray.com/maven</url>
     </repository>
   </repositories>
-
-  <distributionManagement>
-    <snapshotRepository>
```

Comment:
The parent pom is set to `org.apache:apache` and from that we're inheriting all the right settings.

I have followed the instructions at http://www.apache.org/dev/publishing-maven-artifacts.html